### DISCUSSION TITLE
Pretendard Variable 글꼴의 Chrome 렌더링 문제

### DISCUSSION THREAD

**Discussion by lego37yoon** [orioncactus/pretendard#141] [Q&A] [answered]
안녕하세요, CDN 이용하여 Pretendard Variable 다이나믹 서브셋 글꼴 적용 중 글꼴 굵기 관련하여 정상적으로 적용되지 않는 문제 발견하여 문의 드립니다.

현재 배포되는 CSS 파일에는 `local("Pretendard Variable")` 이용하여 글꼴 로딩을 향상하기 위한 내용이 적용되어 있는 것으로 인지하고 있는데요,

Ubuntu 22.04 / Chromium 114 환경에서 Pretendard Variable TTF 파일 설치된 상태로 확인해보니 Chrome 버그로 인하여 로컬에서 글꼴을 읽어올 수 있으면 글꼴 굵기 옵션이 무시되는 현상을 확인하였습니다.

- CDN의 CSS 파일 내용을 그대로 복사 붙여넣은 뒤, local만 제외 시 정상 작동합니다.
- CDN의 CSS 파일 내용을 복사 붙여넣고 변경하지 않거나, `README.md`의 내용과 같이 `@import` 문 사용할 경우, 동일 문제 발생하므로 CDN이나 글꼴 파일 문제는 아니라고 판단했습니다.
- import 하지 않고 내장 글꼴로만 불러오도록 설정하면 문제 없이 잘 작동합니다.
- Firefox 브라우저에서는 어떤 옵션이어도 잘 작동합니다.

이에 따라 local을 의도적으로 제외하고 사용하고자 하는데, 이러한 옵션을 제공해주실 수 있는지

혹은 다이나믹 서브셋 제공에 사용하였던 개별 WOFF2 파일만 CDN을 사용하여 다이나믹 서브셋 CSS 직접 구성하여 프로젝트에 사용해도 라이선스 및 약관 상의 문제가 없는지 문의드립니다.

감사합니다.

**Comment by orioncactus** [comment, marked as answer]
안녕하세요! 저도 살펴보았는데요, 모든 OS에서 동작하는 Chrome에서 말씀하신 것처럼 가변 글꼴이 이미 local에 있을 때 굵기가 정상적으로 표시되지 않는 문제가 나타나는 것을 확인했습니다.

다가오는 일요일 밤에 모든 가변 서브셋에 포함된 local 코드를 제외해 새 버전을 배포하려고 합니다. 혹시나 그동안 local을 제외해 사용하고자 하신다면, Pretendard 레포지터리에 있는 CSS를 임의로 수정하셔도 전혀 문제가 되지 않습니다.

혹시나 다른 문제가 보인다면 언제든지 알려주세요! Chrome에서 발생하는 버그를 알려주시고 Pretendard를 더욱 쾌적하게 제공할 수 있게 도와주셔서 감사드립니다! ദ്ദി•̀.̫•́✧

**Reply by orioncactus** [reply]
적용되었습니다! 이제 CDN을 통해 `@1.3.8` 버전으로 가변 글꼴 문제를 해결한 CSS를 바로 사용하실 수 있습니다 :D

**Reply by lego37yoon** [reply]
감사히 잘 사용하겠습니다 :) 빠르게 대응해주셔서 감사드립니다.
